docs/monitoring-usage.rst

```diff
@@ -278,3 +278,110 @@ follows::
+which similarly requires that its available labels have values specified.
+
+>>> from gcloud import monitoring
+>>> import datetime
+>>> client = monitoring.Client()
+>>> label = LabelDescriptor('response_code', LabelValueType.INT64,
+...    description='HTTP status code')
+>>> metric_descriptor = client.metric_descriptor(
+...    'custom.googleapis.com/my_app/response_count',
+...    metric_kind=MetricKind.CUMULATIVE,
+...    value_type=ValueType.INT64,
+...    labels=[label],
+...    description='Cumulative count of HTTP responses.')
+>>> if not metric_descriptor.exists():
+>>>     metric_descriptor.create()
+>>> # Create a Resource by its resource name, assign labels as dict
+>>> resource = client.resource('gce_instance',
+...    labels={
+...    'instance_id': 'my-instance',
+...    'zone': 'us-central1-f'
+...    })
+>>> # Create a Metric using its metric descriptor parametrized with labels
+>>> metric = client.metric(descriptor=metric_descriptor, labels={
+...    'response_code': 404
+... })
+
+It's also possible to assign labels to Metrics and Resources with keyword arguments::
+
+>>> metric = client.metric(descriptor=metric_descriptor, response_code=404)
+>>> resource = client.resource('gce_instance', instance_id='my-instance', zone='us-central1-f')
+
+Please refer to the `Metrics`_ documentation for more information.
+
+With a Metric and Resource specified, the :class:`~gcloud.monitoring.timeseries.MetricStream`
+can be used to write  :class:`~gcloud.monitoring.timeseries.Point` values.
+:class:`~gcloud.monitoring.timeseries.MetricStream` is a helper class that helps create
+:class:`~gcloud.monitoring.timeseries.TimeSeries` objects with the same Metric and Resource types.
+
+When writing points with :class:`~gcloud.monitoring.timeseries.MetricStream`, you specify
+points containing a value of the type that matches the *value_type* specified in the associated
+:class:`~gcloud.monitoring.metric.MetricDescriptor`. The point must also contain a time interval.
+The point's time interval must be later than any points already created for the same
+:class:`~gcloud.monitoring.timeseries.TimeSeries`
+(where :class:`~gcloud.monitoring.timeseries.TimeSeries` with matching :class:`~gcloud.monitoring.metric.Metric` and :class:`~gcloud.monitoring.resource.Resource` types are considered the same).
+ The end
+time of the interval must not be more than 24 hours in the past or more than five minutes in the
+future::
+
+    >>> start = datetime.utcnow()
+    >>> end = datetime.utcnow() - datetime.timedelta(minutes=5)
+    >>> value = get_num_404s_in_last_five_minutes() # get value from a hypothetical call
+    >>> metric_stream = client.metric_stream(metric, resource)
+    >>> metric_stream.write_point(value, start_time=start, end_time=end) # API call
+
+Stackdriver Monitoring supports several **value types**. A *CUMULATIVE* metric represents a value
```

Comment:
`CUMULATIVE` is a metric kind, not a value type.

Generally, we should try hard to minimize the amount of text here that duplicates the API documentation. It's too difficult to get right, and it's going to be a headache to keep in sync.